src/Skills.jsx

```diff
@@ -128,23 +126,23 @@ class Skills extends Component {
               <Select
                 name="skills"
                 labelKey="name"
-                value={this.state.skill_type}
                 onChange={onSkillChange}
+                value={this.state.currentSkill.skill_type}
                 options={this.state.skills}
               />
               </form>
             </Col>
-            <Button name="AddSkill"
-                    bsStyle="primary"
-                    onClick={this.openSkillModal}>
-              Add Skill
-            </Button>
+            <Button
+              name="AddSkill"
+              bsStyle="primary"
+              onClick={this.openSkillModal}
+              children="Add Skill" />
```

Comment:
Originally, the `Add Skill` child was passed to `Button` as a nested JSX element, as you would see in HTML. You've changed it to being passed as an explicit attribute/prop. This is entirely stylistic, and there is no functional difference, as far as I'm aware. 

Would you please explain why you did this? I do not necessarily prefer either approach over the other, but I think that we should choose one and stick with it, rather than having both styles appear throughout the code base (this probably applies to other things as well, like column width and empty lines between functions, etc...). This approach does seem more explicit.

Would #34 help address this?